ip-messaging/rest/members/remove-member/remove-member.4.x.rb

```diff
@@ -9,5 +9,5 @@
 # Remove the member
 service = ip_messaging_client.services.get('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
```

Comment:
@hortega I assume this feature already exists in twilio-ruby v4, so, is the problem that not all these samples are supported by v4?